### ISSUE TITLE
Incorrect information in add_subparser() docs. 

### ISSUE THREAD

**Issue by JCrane512** [python/cpython#95377]
# Documentation

When setting the flag `required=true` for the function `ArgumentParser.add_subparsers()` the setting `dest=" some description"` also needs to be set or the parser throws an unhelpful error when the user fails to select a subparser. 

This is not covered in the documentation. 

### Linked PRs
* gh-95490

**Comment by MonadChains** [comment]
I agree. The documentation is not fully clear, and adding a line to clarify it could be beneficial, especially given the complexity of the module.

**Comment by StanFromIreland** [comment]
CC @savannahostrowski

**Comment by savannahostrowski** [comment]
It’s unclear which Python version this was originally reported against. The underlying problem was fixed in https://github.com/python/cpython/pull/3680 and backported to Python 3.9 and 3.10. When no subparser is selected, argparse tells the user that one is required: `error: the following arguments are required: {foo,bar}`

It does not reproduce on any branch in the current support window so I think this can be closed.
